labelbox/data/annotation_types/label.py

```diff
@@ -37,7 +40,7 @@ class Label(BaseModel):
         extra: additional context
     """
     uid: Optional[Cuid] = None
-    data: Union[VideoData, ImageData, TextData, TiledImageData]
+    data: DataType
```

Comment:
is this a generic pydantic type? sorry pydantic documentation links do not work for me for some reason

Reply:
I've defined this type on line 19 above since the list was getting too long.